CHANGELOG.md

```diff
@@ -5,69 +5,68 @@ PyO3 versions, please see the [migration guide](https://pyo3.rs/master/migration
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.12.0] - 2020-10-12
```

Comment:
This should probably be 2020-09-12! (Or 13, depending on timezones)

Reply:
Oh haha oops, I just released! I'll merge this now and then fix on master :)